<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="wTerm" name="outputPane">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>Baud Rate</string>
          </property>
          <property name="buddy">
           <cstring>baudBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="baudBox">
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port Name</string>
          </property>
          <property name="buddy">
           <cstring>portnameBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="portnameBox">
          <property name="statusTip">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Data Bits</string>
          </property>
          <property name="buddy">
           <cstring>databitsBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="databitsBox">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Parity</string>
          </property>
          <property name="buddy">
           <cstring>parityBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parityBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mark</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stop Bits</string>
          </property>
          <property name="buddy">
           <cstring>stopbitsBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stopbitsBox">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Direction</string>
          </property>
          <property name="buddy">
           <cstring>directionBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="directionBox">
          <item>
           <property name="text">
            <string>Input</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Output</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>In/Out</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Flow Control</string>
          </property>
          <property name="buddy">
           <cstring>flowcontrolBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="flowcontrolBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hardware</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Software</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>74</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="sendLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendLineButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openPortButton">
        <property name="text">
         <string>Open Serial Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePortButton">
        <property name="text">
         <string>Close Serial Port</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>wTerm</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">wterm.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
